images/ceph/Makefile

```diff
@@ -65,12 +65,13 @@ do.build: generate-csv-ceph-templates
 	@mkdir -p $(TEMP)/rook-external/test-data
 	@cp ../../cluster/examples/kubernetes/ceph/create-external-cluster-resources.* $(TEMP)/rook-external/
 	@cp ../../cluster/examples/kubernetes/ceph/test-data/ceph-status-out $(TEMP)/rook-external/test-data/
-	@if [ ! "$(INCLUDE_CSV_TEMPLATES)" = "" ]; then\
-		cp -r ../../cluster/olm/ceph/templates $(TEMP)/ceph-csv-templates;\
-	else\
-		mkdir $(TEMP)/ceph-csv-templates;\
-	fi
-	@cd $(TEMP) && $(SED_CMD) 's|BASEIMAGE|$(BASEIMAGE)|g' Dockerfile
+ifeq ($(INCLUDE_CSV_TEMPLATES),true)
```

Comment:
Move the check `if [ ! "$(INCLUDE_CSV_TEMPLATES" = ""]` checks to here so that lower level targets can remove the check and look cleaner.